tests/molecules/C2_pp/CMakeLists.txt

```diff
@@ -77,6 +77,31 @@ if(NOT QMC_CUDA)
       C2_PP_MSDJ_LONG # VMC
     )
 
+    if(NOT QMC_MIXED_PRECISION)
+      list(APPEND DET_C2_PP_MSDJ_PARAM CIcoeff_4 0.29045551 0.00001)
+      list(APPEND DET_C2_PP_MSDJ_PARAM ud_4 0.4797811594 0.00001)
```

Comment:
Where did these values come from? Are they the current reported values or the result of some independent evaluation?